src/components/ButtonModal.stories.tsx

```diff
@@ -91,3 +91,17 @@ export function ButtonModalWithChipAndTooltip() {
     />
   );
 }
+
+export function ButtonModalWithSecondaryText() {
```

Comment:
**nit:** I don't think this story is necessary. I get that this is _why_ we build out this new Button variant. But I don't think it is worth us doing chromatic diffs against, as the existing ButtonModal using a Button story + the updated Button stories should cover the same case overall. 
We can definitely keep this, but I'd recommend we (disable it from chromatic snapshots)[https://www.chromatic.com/docs/ignoring-elements#ignore-stories], just to be conscience of increasing our Chromatic costs.

Reply:
thank you!!